webapp/src/index.tsx

```diff
@@ -27,20 +27,20 @@ export default class Plugin {
         registry.registerUserSettings?.({
             id: manifest.id,
             icon: `${serverRoute}/plugins/${manifest.id}/public/msteams-sync-icon.svg`,
-            uiName: 'MS Teams Sync',
+            uiName: manifest.name,
             sections: [{
                 settings: [{
                     name: 'primary_platform',
                     options: [
                         {
-                            text: 'Mattermost will be my primary platform',
+                            text: 'Mattermost',
                             value: 'mm',
-                            helpText: 'You will need to disable notifications in Microsoft Teams to avoid duplicates. **[Learn more](http://google.com)**',
+                            helpText: 'You will get notifications in Mattermost for synced messages and channels. You will need to disable notifications in Microsoft Teams to avoid duplicates. **[Learn more](http://google.com)**',
                         },
                         {
-                            text: 'Microsoft Teams will be my primary platform',
+                            text: 'Microsoft Teams',
                             value: 'teams',
-                            helpText: 'Notifications in Mattermost will be muted for linked channels and DMs to prevent duplicates. Unread statuses in linked channels and DMs will also be disabled in Mattermost. **[Learn more](http://google.com)**',
+                            helpText: 'Notifications in Mattermost will be muted for linked channels and DMs to prevent duplicates. You can unmute any linked channel or DM/GM if you with to receive notifications. **[Learn more](http://google.com)**',
```

Comment:
```suggestion
                            helpText: 'Notifications in Mattermost will be muted for linked channels and DMs to prevent duplicates. You can unmute any linked channel or DM/GM if you wish to receive notifications. **[Learn more](http://google.com)**',
```